### PR TITLE
WIP - added example with new router

### PR DESCRIPTION
**Pull request by sumitarora**

### CODE REVIEW

File: example-apps/kitchen-sink-new-router/source/components/router/start.ts [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+      component: StartMain
+    }, {
+      path: 'child',
+      component: StartChild
+    }, {
+        path: 'router-data2/:name/:message',
+        component: RouterData2
+      }]
+  }
+  // { aux: '/auxcomp', component: AuxComp },
```
Comment (igor-ka) [inline example-apps/kitchen-sink-new-router/source/components/router/start.ts]: Can we remove the commented out code?

Comment (sumitarora) [inline example-apps/kitchen-sink-new-router/source/components/router/start.ts]: Its still WIP will do once ready

**Comment by ericjim** [comment]
I'll look into fixing the demo under `/start`

**Comment by igor-ka** [comment]
I think that the example app update should happen in conjunction with the support for the new router. This way we can just update the kitchen-sink app w/o adding a separate folder.

**Comment by sumitarora** [comment]
@igor-ka I added a different one so we can have both versions of router working and make Augury work with both of them. As I lot of apps are still using old router.

**Comment by sumitarora** [comment]
Closing this continued with Augury update here: https://github.com/rangle/augury/pull/471

**Pull request closed without merging**